simple-base-sequence.puml

```diff
@@ -0,0 +1,24 @@
+@startuml
+
+participant "Consumer" as consumer
+participant "Association Register" as association
+participant "Authorization Register (owner)" as ownerAr
+participant "Resource (owner)" as ownerApi
+
+consumer -> ownerApi : /token (get authentication token)
+ownerApi -> ownerApi : check client assertion
+ownerApi -> association : /token (get authentication token)
+association -> association : check client assertion
+association -> association : check owner still compliant
+association -> ownerApi : OK: token
+ownerApi -> association : /party (with token get consumer party information)
+association -> association : check token
+association -> ownerApi : OK: party info
+ownerApi -> ownerApi : check consumer party info for compliance
+ownerApi -> consumer : OK: token
+
+consumer -> ownerApi : /some-resource (with token)
+ownerApi -> ownerApi : check token
+ownerApi -> consumer : OK: data
+
+@enduml
```

Comment:
In dit diagram staat niet dat de ownerApi bij het ownerAr een check doet. ownerApi zou bij ownerAr delegation-evidence moeten opvragen

Reply:
Er zijn 3 flows die ik beschrijf (zie ook cookbook recepten):
- service consumer stuurt delegation evidence mee
- service provider haalt zelf delegation evidence op
- service provider laat AR links liggen en doet ad hoc authorization

Dit diagram toont de derde flow. Het `base sequence` diagram toont de eerste flow.